### PR TITLE
[WebDriver BiDi] Add `input.setFiles` tests

### PR DESCRIPTION
**Pull request by jrandolf**

### CODE REVIEW

File: webdriver/tests/bidi/input/set_files/set_files.py [hunk 1]
```diff
@@ -0,0 +1,273 @@
...
+    load_static_test_page,
+    get_element,
+):
+    await load_static_test_page(page="files.html")
+
+    element = await get_element("#input")
+
+    await bidi_session.input.set_files(
+        context=top_context["context"],
+        element={"sharedId": element["sharedId"]},
```
Comment (Lightning00Blade) [inline webdriver/tests/bidi/input/set_files/set_files.py:21]: I think we can do this everywhere.
```suggestion
        element=element,
```

**Review by Lightning00Blade — COMMENTED** [review]
Left some comments, it would be nice to also add the other invalid test as well.

File: webdriver/tests/bidi/input/set_files/invalid.py [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+
+
+@pytest.mark.parametrize(
+    "files",
+    [
+        ["path/to/noop.txt"],
+        [],
+    ],
+)
+async def test_set_files_missing_element(
```
Comment (OrKoN) [inline webdriver/tests/bidi/input/set_files/invalid.py:15]: the invalid tests should follow the `test_params_PARAMNAME_test_name` and `test_params_PARAMNAME_invalid_type` and `test_params_PARAMNAME_invalid_value` for scheme tests

File: webdriver/tests/support/html/files.html [hunk 3]
```diff
@@ -0,0 +1,30 @@
+<html>
```
Comment (OrKoN) [inline webdriver/tests/support/html/files.html:1]: include doctype please

File: webdriver/tests/bidi/input/set_files/invalid.py (hunk 2 above)
Comment (jrandolf) [reply webdriver/tests/bidi/input/set_files/invalid.py]: Could you elaborate `test_params_PARAMNAME_invalid_type` and `test_params_PARAMNAME_invalid_value`

**Review by OrKoN — APPROVED** [review]
LGTM

**Review by Lightning00Blade — APPROVED** [review]

**Comment by jrandolf** [comment]
@web-platform-tests/admins Please force merge.

**Pull request merged**